File: zxing-android-embedded/src/com/journeyapps/barcodescanner/CompoundBarcodeView.java
package com.journeyapps.barcodescanner;

import android.content.Context;
import android.util.AttributeSet;

/**
 * Used as an alias for DecoratedBarcodeView, for backwards-compatibility.
 */
public class CompoundBarcodeView extends DecoratedBarcodeView {
    public CompoundBarcodeView(Context context) {
        super(context);
    }

    public CompoundBarcodeView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public CompoundBarcodeView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: zxing-android-embedded/src/com/journeyapps/barcodescanner/CompoundBarcodeView.kt
package com.journeyapps.barcodescanner

import android.content.Context
import android.util.AttributeSet

/**
 * Used as an alias for DecoratedBarcodeView, for backwards-compatibility.
 */
class CompoundBarcodeView : DecoratedBarcodeView {
    constructor(context: Context) : super(context)
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int = 0) : super(context, attrs, defStyleAttr)
}